cartridge-scm-1.rockspec

```diff
@@ -25,7 +25,6 @@ external_dependencies = {
 build = {
     type = 'cmake',
     variables = {
-        version = 'scm-1',
```

Comment:
Why do you remove it? Seems it isn't related to your patch.

Reply:
Removing this line eliminates a warning:

```text
CMake Warning:                                                                                                                                                                                             
  Manually-specified variables were not used by the project:                                                                                                                                               
                                                                                                                                                                                                           
    version    
```

Reply:
Good